example/exporter/kubelet-servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+# You need to have "--kubelet-object=kube-system/kubelet" as a prometheus-operator arg
```

Comment:
@brancz thanks. I am wondering if would be better use a more generic definition that match all `k8s-app` with `https-metrics` endpoints

```yaml
apiVersion: monitoring.coreos.com/v1alpha1
kind: ServiceMonitor
metadata:
  name: k8s-apps-https
  labels:
    k8s-apps: https
spec:
  jobLabel: k8s-app
  selector:
    matchExpressions:
    - {key: k8s-app, operator: Exists}
  namespaceSelector:
    matchNames:
    - kube-system
  endpoints:
  - port: https-metrics
    interval: 15s
    scheme: https
    tlsConfig:
      caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
      insecureSkipVerify: true
    bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
```